charts/shoot-core/components/charts/nodelocal-dns/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Hm, what's the purpose of this service? It points to CoreDNS, or? How is it different from https://github.com/gardener/gardener/blob/master/charts/shoot-core/components/charts/coredns/templates/coredns-service.yaml?

Reply:
this is to avoid DNS loops, the service IP needs to be different.